src/base/system_parameter.c

```diff
@@ -6488,6 +6507,12 @@ static KEYVAL server_type_words[] = {
   {server_type_config_to_string (server_type_config::PAGE), (int) server_type_config::PAGE},
   {server_type_config_to_string (server_type_config::SINGLE_NODE), (int) server_type_config::SINGLE_NODE}
 };
+
+static KEYVAL transaction_server_type_words[] = {
```

Comment:
```suggestion
static constexpr KEYVAL transaction_server_type_words[] = {
```
All mappings in this file can be `constexpr` - but maybe is beyond the scope of this PR:
```
static constexpr KEYVAL boolean_words[] = {
static constexpr KEYVAL er_log_level_words[] = {
static constexpr KEYVAL isolation_level_words[] = {
static constexpr KEYVAL pgbuf_debug_page_validation_level_words[] = {
static constexpr KEYVAL null_words[] = {
static constexpr KEYVAL ha_mode_words[] = {
static constexpr KEYVAL ha_server_state_words[] = {
static constexpr KEYVAL ha_log_applier_state_words[] = {
static constexpr KEYVAL compat_words[] = {
static constexpr KEYVAL check_peer_alive_words[] = {
static constexpr KEYVAL query_trace_format_words[] = {
static constexpr KEYVAL fi_test_words[] = {
static constexpr KEYVAL ha_repl_filter_type_words[] = {
static constexpr KEYVAL tde_algorithm_words[] = {
static constexpr KEYVAL server_type_words[] = {
```

Reply:
i think a small issue is best suited for this change.

Reply:
Please limit the review suggestions in the purpose of the patch, or to things that may have a meaningful impact. The legacy code is still 99% C and there are thousands of examples like this, that may be modernized and aligned to new C++ standards.